collect_app/src/main/java/org/odk/collect/android/logic/FormController.java

```diff
@@ -409,13 +409,11 @@ private boolean indexIsInFieldList(FormIndex index) {
             return groupIsFieldList(grp.getIndex());
         } else if (event == FormEntryController.EVENT_GROUP) {
             return groupIsFieldList(index);
-        } else if (event == FormEntryController.EVENT_REPEAT) {
-            return repeatIsFieldList(index);
-        } else {
-            // right now we only test Questions and Groups. Should we also handle
-            // repeats?
-            return false;
-        }
+        } else
```

Comment:
Let's leave this one as it was because the comment makes more sense in that context. Also, we should never have a conditional block without curly braces (https://google.github.io/styleguide/javaguide.html#s4.1-braces). Now that we have checkstyle and lint, make sure to run the tasks before submitting a PR!